### DISCUSSION TITLE
Failure "Cannot GET /link/9acc7Bf002B8D3cc9cA1ddca2BBBAe97"

### DISCUSSION THREAD

**Discussion by Jaques-Ludwig** [hass-emulated-hue/core#209] [Q&A]
I tried to connect Hue Essentials with hass-emulated-hue.

The App asked to push the button on the bridge and I get a message in Home Assistant. But if I use the link in Home Assistant I get the message in my browser:

"Cannot GET /link/9acc7Bf002B8D3cc9cA1ddca2BBBAe97"

Can someone help?

Thanks,

Stephan

**Comment by Jaques-Ludwig** [comment]
Ok, I started all new (Home Assistant, Emulated Hue), deleted App-Data of Hue Essentials. 
I saw, that hassio_observer in mei container list was on Port 80. I switched it to 82 and now all works.

Stephan

**Reply by alexyao2015** [reply]
Be aware that using ports other than 80 and 443 will result in unexpected behavior in some apps. As additionally, not using host networking results in discovery not functioning.
